src/awsexplorer/regionNode.ts

```diff
@@ -107,6 +115,9 @@ export class RegionNode extends AWSTreeNodeBase {
         const partitionId = this.regionProvider.getPartitionId(this.regionCode) ?? defaultPartition
         const childNodes: AWSTreeNodeBase[] = []
         for (const { serviceId, createFn } of serviceCandidates) {
```

Comment:
if this for-loop was less clever we could have a more ergonomic way to define "conditional" items.
```suggestion
        for (const service of serviceCandidates) {
            if (service.when === undefined || !service.when()) {
                continue
            }
```

Then in `serviceCandidates` we can define the item like this:

```
    {
        serviceId: 'ec2',
        when: DevSettings.instance.isDevMode(),
        createFn: (regionCode: string, partitionId: string) =>
            new Ec2ParentNode(regionCode, partitionId, new Ec2Client(regionCode)),
    },
```

Reply:
Oh that was just my logic error :) `when` should not be required